src/upload-tests.proj

```diff
@@ -121,8 +125,37 @@
   <!-- main Entrypoint -->
   <Target Name="Build">
     <Message Text="CoreFX-specific Helix test upload project, beginning submission to Helix" />
-    <Message Condition="'$(EnableCloudTest)' == 'false'" Text="... skipping upload / submission due to property 'EnableCloudTest' being set to 'false'" />
+    <Message Condition="'$(EnableCloudTest)' == 'false'" Text="... will skip upload / submission due to property 'EnableCloudTest' being set to 'false'" />
+    <Message Condition="'$(SkipSendToHelix)' == 'true'"  Text="... will skip submission due to property 'SkipSendToHelix' being set to 'true'" />
     <CallTarget Condition="'$(EnableCloudTest)' != 'false'" Targets="CoreFXPreCloudBuild;CompressRuntimeDir;HelixCloudBuild" />
+    <CallTarget Condition="'$(LocalHelixExecution)' == 'true'" Targets="ExecuteLocally" />
+
   </Target>
-  
+
+  <!-- Local Execution entrypoint, can be invoked directly -->
+  <!-- Sample direct command line:
```

Comment:
Can we please get some documentation added to the md files about how to do this?